frontend/packages/dev-console/src/components/import/DeploySection.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { ExpandableSection } from '@patternfly/react-core';
+import { FormikValues } from 'formik';
+import { useTranslation } from 'react-i18next';
+import { AppResources } from '../edit-application/edit-application-types';
+import DeploymentConfigSection from './advanced/DeploymentConfigSection';
+import FormSection from './section/FormSection';
+import ResourceSection from './section/ResourceSection';
+
+type DeploySectionProps = {
+  values: FormikValues;
+  appResources?: AppResources;
+};
+
+export const DeploySection: React.FC<DeploySectionProps> = ({ values, appResources }) => {
+  const { t } = useTranslation();
+  return (
+    <FormSection title={t('devconsole~Deploy')} fullWidth>
+      <ResourceSection />
+
+      <ExpandableSection toggleText={t('devconsole~Show Deployment configuration')}>
```

Comment:
Does it make sense to align this naming with "Show advanced Routing options"?

```suggestion
      <ExpandableSection toggleText={t('devconsole~Show advanced Deployment option')}>
```